src/register/RegistrationPage.jsx

```diff
@@ -228,6 +228,8 @@ const RegistrationPage = (props) => {
 
   useEffect(() => {
     if (registrationResult.success) {
+      sendTrackEvent('edx.bi.user.account.registered.client', {});
+
       // Optimizely registration conversion event
```

Comment:
Should we not remove the optimizely event and mape the newly added event into the optimizely?

Reply:
If we do it in the same PR, we will lose data. I will do it as a followup.